### DISCUSSION TITLE
HELP!!! "Please add EOL settings" after Configuration Reset - PLEASE HELP!!!!

### DISCUSSION THREAD

**Discussion by Alivelton** [cedricp/ddt4all#946] [Q&A]
I need help with my car and a fluence

**Comment by shrlnm** [comment]
@Alivelton Have you tried my recipe https://www.drive2.ru/b/643019689764615344/?

**Reply by Alivelton** [reply]
Pyren3 does not open

**Reply by Alivelton** [reply]
And when pyren presses update it says in gitlab connection

**Reply by Alivelton** [reply]
![Screenshot_2023-11-17-22-31-06-849_com.brave.browser.jpg](https://github.com/cedricp/ddt4all/assets/150972135/1f7ada85-0499-437c-b08b-ef75107a7fd8)

This part I don't understand

**Reply by morganclio** [reply]
> @Alivelton Have you tried my recipe https://www.drive2.ru/b/643019689764615344/?

@shrlnm  I've same problem . My radio is displaying the yellow triangle "unconfigured radio"  For your solution do I need
 cable 1 or
![20250327_131407](https://github.com/user-attachments/assets/68e97096-041d-4fb0-babf-7d6770793a31)
cable 2 
![61vMK-0U+qL](https://github.com/user-attachments/assets/ce2c4940-6d3c-4872-ac70-582a3bba1b95)

or both?
I connected laptop with cable 1 to my MK4 Clio but i don't see this option below. When I scan for ECUs i don't have any AUDIO/RADIO if I understand this correctly.  I have listed things like ABS,GEARBOX,AIRCON etc but nothing that I could enter to get this below. What Im doing wrong please?
![1000049346](https://github.com/user-attachments/assets/c4629b67-39fa-46ff-a067-ce179bbc434d)

**Reply by AndyIreland** [reply]
@morganclio  - dont forget you have to make adjusments to the wiring of this fly splitter cable 

![image](https://github.com/user-attachments/assets/f112b676-6189-4edc-8583-8785bc0cb223)


to enable to read your rlink through the OBD2 socket otherwise it wont work. 

and has your ELM327 got the required PIC chip in it ?

and yes to read/write rlink1 parameters through OBDII port you have to use both ELM327 and the modified splitter cable there

**Comment by AndyIreland** [comment]
> And when pyren presses update it says in gitlab connection

PyREN = Gitlab = https://gitlab.com/py_ren/pyren
DDT4ALL = Github = https://github.com/cedricp/ddt4all/

**Reply by Alivelton** [reply]
![Screenshot_2023-11-17-22-31-06-849_com.brave.browser.jpg](https://github.com/cedricp/ddt4all/assets/150972135/6cebb476-f4e1-4966-9284-26f7a830f753)

I want help getting here

**Reply by shrlnm** [reply]
Probably your VIN is absent in DB.

**Comment by AndyIreland** [comment]
@morganclio  - have you got a MACRO folder in your PyREN3 there is a rlink1 CMD scrip you could try in that one. 

![image](https://github.com/user-attachments/assets/841aa9a9-c40c-42fd-82b0-7eb6763bb95c)


![image](https://github.com/user-attachments/assets/e94a50d2-cd99-4b24-8c48-b62df093dc05)

**Reply by morganclio** [reply]
> @morganclio - have you got a MACRO folder in your PyREN3 there is a rlink1 CMD scrip you could try in that one.
> 
> ![image](https://private-user-images.githubusercontent.com/49172217/427623369-841aa9a9-c40c-42fd-82b0-7eb6763bb95c.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDMwOTQzMzIsIm5iZiI6MTc0MzA5NDAzMiwicGF0aCI6Ii80OTE3MjIxNy80Mjc2MjMzNjktODQxYWE5YTktYzQwYy00MmZkLTgyYjAtN2ViNjc2M2JiOTVjLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTAzMjclMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwMzI3VDE2NDcxMlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWEzNGFhNjQ5OGI5YWYwNTMxNWZmNjFjYTUwMmMzNGUyNzQ2OWM3ZmUwMDM4NzU3YzJjOTc1NGU1OGMxZWFlOTUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.PkO1yVhRtnzzWLSKVQ3Sm5O3sY6afVd3Tm11CmOsok4)
> 
> ![image](https://private-user-images.githubusercontent.com/49172217/427623968-e94a50d2-cd99-4b24-8c48-b62df093dc05.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDMwOTQzMzIsIm5iZiI6MTc0MzA5NDAzMiwicGF0aCI6Ii80OTE3MjIxNy80Mjc2MjM5NjgtZTk0YTUwZDItY2Q5OS00YjI0LThjNDgtYjYyZGYwOTNkYzA1LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTAzMjclMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwMzI3VDE2NDcxMlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWE1ZmY4MjgzMWYwYzUzOTM1MDIxNzgwNzFiMDBkZWZhZWYxZGFlOWZmZDJlMjI2NGIzYWFjNDQyZGVkZDlkZGEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.w625q9DwWFP1ULSsGSSzSBvphibyGDMk3ghy_-9EYhA)

@AndyIreland I have this in Macro  Rlink2
![macro](https://github.com/user-attachments/assets/c6231023-1bcb-4e44-b8f4-274d833c69f5)

**Reply by AndyIreland** [reply]
here is folder from my pyren3 macro folder - i dont know if it will work for your unit , or whether it was just for mine:
https://we.tl/t-kHxdy3KHew

think is its possible that my VIN number is inputted so at the very least you will have to put in your VIN and CRC into the VIN  box . i dont know if that would work.

**Reply by morganclio** [reply]
@AndyIreland the folder/files is this for the pyren3 software to get the radio working or is it helpful for the option using the dtt4all as I'm now lost what and where I should use ...

**Reply by AndyIreland** [reply]
its a MACRO .CMD script file for using with PyREN3  

![image](https://github.com/user-attachments/assets/e06d4cf2-840e-4f81-98c7-64ce7e79c56c)

**Reply by morganclio** [reply]
So you click the Macro, open the file from Marco folder and follow the steps?

**Reply by AndyIreland** [reply]
yeah I think thats how it works. although when I fixed my EOL message on screen on my rlink 1 in my Zoe EV I didnt even try the CMD Macro way , I managed to sort mine out and get mine working again by inputting the VIN in the box and the CRC and then filling out all the other missing parameters in the boxes then restarted then screen booted fine and beeped through speakers and rlink1 asked me for the PIN password on the screen and then I turned of car , locked it up for 20mins, had something to eat, went out and beeping through speakers wasnt there any more and the rlink1 booted up and worked pefectly.

**Reply by morganclio** [reply]
You were filling all the other parameters in ddt4all if I'm correct yes?

**Reply by AndyIreland** [reply]
yes i did - but you can in Pyren 3 too

**Reply by Furtif** [reply]
> here is folder from my pyren3 macro folder - i dont know if it will work for your unit , or whether it was just for mine: https://we.tl/t-kHxdy3KHew
> 
> think is its possible that my VIN number is inputted so at the very least you will have to put in your VIN and CRC into the VIN box . i dont know if that would work.

You should make a PR (pull request) at https://gitlab.com/py_ren/pyren/-/tree/pyren3/pyren3/macro, instead of having something just for yourself and sharing independently, the fact of creating the PR does not take away whoever made it or the credits, the only thing that remains is that everything that is part of a free and open source program is not diverted to other interests.

**Reply by morganclio** [reply]
@AndyIreland  , @Furtif   Would any of you know if this file could remove/not activate the RS monitor?  The reason I'm asking is that my Clio is an RS and it had the RS monitor...

**Reply by Renotech1** [reply]
> @AndyIreland  , @Furtif   Would any of you know if this file could remove/not activate the RS monitor?  The reason I'm asking is that my Clio is an RS and it had the RS monitor...

I think you can disable rs monitor manually in ddt 👌

**Reply by morganclio** [reply]
> > @AndyIreland  , @Furtif   Would any of you know if this file could remove/not activate the RS monitor?  The reason I'm asking is that my Clio is an RS and it had the RS monitor...
> 
> I think you can disable rs monitor manually in ddt 👌

@Renotech1  I don't want to disable :) I want to keep it. Andy shared his Macro file with me so I can use it with thy Pyren3 but his file is not from an RS that's why my question come. Should I use it or should I look for someone with with Clio RS with the monitor that could share the file?

**Reply by Furtif** [reply]
> @AndyIreland , @Furtif Would any of you know if this file could remove/not activate the RS monitor? The reason I'm asking is that my Clio is an RS and it had the RS monitor...

I don't know, I can't know everything, I just said, that instead of having something that serves pyren and other users (more or less private) to propose in the respective repo, nothing more.

**Reply by AndyIreland** [reply]
> @AndyIreland , @Furtif Would any of you know if this file could remove/not activate the RS monitor? The reason I'm asking is that my Clio is an RS and it had the RS monitor...

@morganclio  - you are most probably right, it wont include the RS monitor and you will have to put in the parameters into these boxes - but i have not got a clue what you have write into them sorry
![image](https://github.com/user-attachments/assets/55e16e64-eb2d-4904-902c-97df62909822)

**Reply by morganclio** [reply]
@AndyIreland   so it looks like my ELM327 doesn't work...  I need a new one. Do you think any of them will do the job? 
https://www.amazon.co.uk/DollaTek-Diagnostic-Scanner-Reader-bluetooth/dp/B09KN3J6V6/ref=sr_1_6?crid=47JBBWBDCCAK&dib=eyJ2IjoiMSJ9.p6KzXtYgpnXpCwTqm3ZyQzxnnky1R0uhDJ0zNUKxqp3jObFEzmucxB7OIznzZ5NP8VnxAeX7fvFnUDyjQhonh1rUXBJXuitY_pSal21lAOPJE0eOJGyH4E4JiTZ1ozcWrOU0MJZCgQId2NL_RbDGlto7mX8kTXELa9-nkJcL6a91Hpc8m--VRwy7Ha5g-3RHDwRnhc0jzpNGO-LwYW0_ofso7wRKeSeSzQyAzOLWauRo44SS7F1UEg-E4_GD7gwipeH9hYQ9lbrT_wdfXbfhyCLnIwkeMZBkG5d1Kb920zPv1ImXyquBjbxFzIGWEUtGzI9qM9lIdubopBJyV1DLmqNVSzy8zLid_jjgzdejWWL3UPNgLftD4hi3LSmtFUskI_K47IJ_5HKr24eQ5dmTY5N7w3Ca-OaaWXEcvVZBDbjpSXxEy4HOvEw0xviqqwTs.ObUI4STWsPhR45nQo2YFKJbOIeZBYTjBk51VM-qqGjU&dib_tag=se&keywords=dollatek+elm327&qid=1743237637&sprefix=dollatek+elm327%2Caps%2C73&sr=8-6

or

https://www.amazon.co.uk/DollaTek-bluetooth-Scanner-Diagnostic-PIC18F25K80/dp/B09SZ7HZNW/ref=pd_ci_mcx_pspc_dp_d_2_i_1?pd_rd_w=hTMar&content-id=amzn1.sym.707a350c-f6d8-4abf-9861-74c140760f91&pf_rd_p=707a350c-f6d8-4abf-9861-74c140760f91&pf_rd_r=RBESV71EJHH5ZCM2NCJS&pd_rd_wg=oxmJ7&pd_rd_r=b0ce0ce6-96bb-4765-a402-2cb85008947b&pd_rd_i=B09SZ7HZNW
or
https://www.amazon.co.uk/Yalgdlen-elm327-Adapter-PIC18F25K80-Scanner/dp/B0DFQ39TRY/ref=sr_1_3_sspa?dib=eyJ2IjoiMSJ9.QxuazZCAtri4JIqhZC-ePCTULY4-vECAAKmStYUiE-BhLtYiGTkLQYLKnrGxTgIn56WPBxi0z4ubZNai2Pf1dMJLjiIo245dtqZOoajQGXKXtfW-tl0YkrwYszezF6OTF3HcMBdFZDZA2il8KHa-lyFR2649sHLlnFgOMMvlodaG2ntbJ9kLg17_nmP_T6U9GS1zeuUxOkpANcpE_oA6dGG-Dq0bsQCfjBq_YVYM3gyFN9omjOwqiTPoZn-owMmyQGmB-ZsjJqmJh_o6bJiVzairlyuYP4GYbJgatJ6fGuQ.nO4_z6o1AuTy1oMnSOfjX8JQClGP9RBab53PG4qWO8A&dib_tag=se&keywords=PIC18F25K80&qid=1743237833&sr=8-3-spons&sp_csd=d2lkZ2V0TmFtZT1zcF9hdGY&psc=1
